.codesandbox/templates/vanilla/src/testcases/guidelines.ts

```diff
@@ -0,0 +1,11 @@
+import * as fabric from 'fabric';
+import { initAligningGuidelines } from 'fabric/extensions';
```

Comment:
@zhe-he this is how i want to import the extensions like the guidelines.
I think we can refine this so that splitting and node/browser are still possible cdn links

Reply:
Yes, this file is not used anywhere, and you can delete it. In the original alignment version, there is a parameter called aligningLineOffset, and I found that its use case is completely ineffective in the new version. Later, I forgot to delete this file, and the offset property in AligningLineConfig should also be removed.